workflow/scripts/merge_cnv_json.py

```diff
@@ -106,21 +106,27 @@ def get_vaf(vcf_filename: Union[str, bytes, Path], skip=None) -> Generator[tuple
         skip = []
     vcf = pysam.VariantFile(str(vcf_filename))
     for variant in vcf.fetch():
-        if variant.chrom in skip:
+        chrom = variant.chrom
+        if not chrom.startswith("chr"):
+            chrom = f"chr{chrom}"
+        if chrom in skip:
```

Comment:
I don't think it should be up to this function to modify the chromosome names. If the chromosome names are mismatching between different files, then that is probably due to different references being in use, e.g. one type of nomenclature in the list of chromosomes to skip and another one for the variant calls/ratios/segments. Or am I missing something?

Reply:
Jumble generates files without chr regardless of reference genome used.

Reply:
Got it, I will modify the Jumble test data and see how it looks.

Reply:
After updating the test files I tried to revert the commit introducing the changes to this file, and everything works as it should in the integration test. Was this an attempt to fix the issue with the `chr` prefix missing in the Jumble files, or was it something else?

Reply:
The merge_cnv_json rule takes in cnv vcf-files from the different callers. One of these is from jumble and when they are compared using a dict I get key error as the chromosome names are differing between the callers.

input:
  filtered_cnv_vcfs=get_filtered_cnv_vcf,
  cnv_vcfs=get_unfiltered_cnv_vcf,

Reply:
Ok, I understand. At the moment the integration test only considers a merged VCF from SVDB, but the Jumble output is then not part of this?

Reply:
I guess that wouldn't matter though since the chromosome names would still be different unless SVDB corrects that somehow. Should I modify the CNV VCF in the integration test so that the Jumble variants don't have the `chr` prefix?

Reply:
yes please!